src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype100.cs

```diff
@@ -199,51 +199,6 @@ protected override void SaveProperties(Database db)
             {
                 db.EncryptionEnabled = this.currentState.encryptionEnabled;
             }
-
-            // Check if we support database scoped configurations in this database. Since these were all added at the same time,
-            // only check if MaxDop is supported rather than each individual property.
-            if (db.IsSupportedProperty("MaxDop"))
```

Comment:
Why did this code get removed? I'm not seeing it moved to anywhere else.

Reply:
Since databaseScopedConfigurations are listed from the databse.DatabaseScopedCofigurationCollections, having these here has no use, as I don't see these are using anywhere. And when we have new dsc props, these old 4 properties are being overriding the new big list of dsc properties which causes an issue. So, removed them from saving properties.

Reply:
If these existed in 100 shouldn't they stay here though?

Reply:
Gotcha, that explanation makes sense to me. Thank you!